src/content/index.ts

```diff
@@ -20,4 +26,12 @@ export namespace Content {
   export function getSpace(): Promise<Space> {
     return getClient().getSpace();
   }
+
+  export function getAsset(id: string, query: any): Promise<Asset> {
+    return getClient().getAsset(id);
+  }
+
+  export function getAssets(query: any): Promise<ContentfulCollection<Asset>> {
```

Comment:
same as here

Reply:
Add query to getClient().getAssets(query) since I would assume we would need some assets collection by query, but not all